frontend/packages/console-metal3/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@console/metal3",
```

Comment:
I'd suggest to follow these conventions:

- package path
  - should include the `plugin` suffix to denote the purpose of the package
  - should not include the `console` prefix (`console-xxx` is reserved for core Console packages)
  - proposed value: `packages/metal3-plugin`

- package name
  - `@console` scope is required due to monorepo and tooling
  - proposed value: `@console/metal3-plugin`

Later on, Metal3 team could add & own other packages as necessary, e.g. `packages/metal3-components` etc.

Reply:
Done